### ISSUE TITLE
test on bird dataset

### ISSUE THREAD

**Issue by steph730** [defog-ai/sqlcoder#18]
i test sqlcoder model on BIRD dataset,only achieve .15 accuracy, anyone have ever test it?

**Comment by rishsriv** [comment]
The BIRD dataset is in SQLite, while SQLCoder is optimized for Postgres. You will have to fine tune it for SQLite before testing on BIRD for best results

**Comment by steph730** [comment]
> The BIRD dataset is in SQLite, while SQLCoder is optimized for Postgres. You will have to fine tune it for SQLite before testing on BIRD for best results

thank you! and how to finetune sql coder on sqlite, is there any open source framework or project?

**Comment by lihaoyang-ruc** [comment]
Hi! @steph730,

We have fine-tuned CodeS (our pre-trained LLM specifically optimized for SQL generation) on Bird, and it has shown impressive performance on Bird's development set, surpassing even GPT-4. The fine-tuned checkpoints are now available for download and immediate use!

For more details, you can check [CodeS](https://github.com/RUCKBReasoning/codes).
